From 6329526199674ddda8d8396a66072890b244972b Mon Sep 17 00:00:00 2001
From: Maxim Shpak <maxim.shpak@posteo.uk>
Date: Tue, 30 Mar 2021 18:13:39 +0300
Subject: [PATCH 14/85] Run systemd service after network is available (unit
 file fix)

There is a race condition between shadowsocks-libev.service and
systemd-resolved.service after reboot/on boot. The shadowsocks
service tries to start before the dns service is started properly
and fails showing the corresponding errors in the logs.
---
 debian/shadowsocks-libev.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/shadowsocks-libev.service b/debian/shadowsocks-libev.service
index 9de8765..0aa4862 100644
--- a/debian/shadowsocks-libev.service
+++ b/debian/shadowsocks-libev.service
@@ -12,6 +12,7 @@
 Description=Shadowsocks-libev Default Server Service
 Documentation=man:shadowsocks-libev(8)
 After=network-online.target
+Wants=network-online.target
 
 [Service]
 Type=simple
-- 
2.25.1

